.clang-format

```diff
@@ -66,7 +66,9 @@ ForEachMacros:
   - Q_FOREACH
   - BOOST_FOREACH
   - TAILQ_FOREACH
-  - WT_CELL_FOREACH_ADDR
+  - WT_CELL_FOREACH
+  - WT_CELL_FOREACH_ADDR 
+  - WT_CELL_FOREACH_END
```

Comment:
Ah, my bad. Can you remove these `END` ones and run `./s_all` again?
Those just look like C statements so they don't need to be formatted as loops.